### ISSUE TITLE
Cursor does not automatically hide on video playback with flash

### ISSUE THREAD

**Issue by andymartinwork** [electron/electron#7307]
* Electron version: 1.3.4
* Operating system: OSX, Windows

I can't reopen #6357 so I will post a new issue.

It looks like when playing a flash based video using the pepperflash plugin and opening fullscreen, the video controls hide but the mouse cursor does not. Perhaps something is preventing the cursor from hiding?

**Comment by MarshallOfSound** [comment]
@andymartinwork If you provide the example that was requested on #6357 we can reopen it.  But without that example we can't reproduce the issue.

Closing this out as a duplicate, just comment on #6357 with a reproducible example and it will be reopened 👍

**Comment by andymartinwork** [comment]
As an example, do you mean with code? I can try to make a small example app if you want.

**Comment by MarshallOfSound** [comment]
@andymartinwork Either a code example that reproduces the issue or a website that contains a video that demonstrates the problem.  Websites like YouTube appear to operate correctly

**Comment by andymartinwork** [comment]
Youtube is not flash based, so would not be affected by this issue.

**Comment by MarshallOfSound** [comment]
@andymartinwork Ah, the original issue only referenced a generic "video".  I believe this was inferred as the HTML5 video element.  Please provide an example demonstrating the issue with flash 👍

**Comment by andymartinwork** [comment]
@MarshallOfSound 

I created a demo app here:

https://github.com/andymartinwork/electron-flash-issue

It's Mac OS only at the moment, but I can add Windows if you want. The issue is there too.

Run npm start and click the fullscreen button on the video. Then notice how the mouse cursor doesn't disappear like it does on other browsers.

I updated it to the latest version of flash just to check that wasn't the issue, but things were the same.

Having a good experience on music videos is really important for us at TIDAL!
